tsconfig.json

```diff
@@ -14,6 +14,8 @@
     "resolveJsonModule": true
   },
   "exclude": [
+    "**/*.spec.ts",
```

Comment:
This will prevent VS Code from being able to provide IntelliSense for imports and global types in `.spec` and `.server` files.
If we want to exclude them from the final bundle, maybe we should add them to the `.npmignore` if they're not already there?

Reply:
I have reverted this, no need to make this change anymore.